sorc/ncep_post.fd/CALHEL3.f

```diff
@@ -0,0 +1,480 @@
+!> @file
```

Comment:
@EdwardColon-NOAA This new routine has similarity with CALHEL2.f. Will you consider to unify them in the future?